spec.html

```diff
@@ -35470,6 +35733,16 @@ <h1>RegExp Records</h1>
                 <td>a Boolean</td>
                 <td>indicates whether *"u"* appears in the RegExp's flags</td>
               </tr>
+              <tr>
+                <td>[[UnicodeSets]]</td>
+                <td>a Boolean</td>
+                <td>indicates whether *"v"* appears in the RegExp's flags</td>
+              </tr>
+              <tr>
+                <td>[[EitherUnicode]]</td>
```

Comment:
I don't like having a record field that duplicates info. Is this just so that tests of both `[[Unicode]]` and `[[UnicodeSets]]` can read better? If so, I'd prefer to introduce an AO for that instead of something persistent.

Reply:
> Is this just so that tests of both `[[Unicode]]` and `[[UnicodeSets]]` can read better?

Yes.

Reply:
PTAL: https://github.com/mathiasbynens/ecma262/pull/22/files

Reply:
Merged.